examples/siro_sandbox/sandbox_app.py

```diff
@@ -290,6 +291,12 @@ def _reset_environment(self):
         self._num_busy_objects = None  # currently held by non-gui agents
 
         sim = self.get_sim()
+
+        # recompute the largest indoor island id whenever the sim backend may have changed
+        self._largest_island_idx = get_largest_island_index(
```

Comment:
@eundersander I think this is the right place to update the navmesh, but please check me.
I'm hoping the simulator is initialized with a fresh scene and all navmesh processing is done.
I also assume sim here is a RearrangeSim instance.